docs/content/latest/deploy/multi-dc/async-replication.md

```diff
@@ -102,6 +102,42 @@ Once you have set up replication, load data into the source universe as follows:
 
 When completed, proceed to [Verifying Replication](#verifying-replication).
 
+## Verifying Replication
+
+You can verify replication by stopping the workload and then using the `COUNT(*)` function on the yugabyte-target to yugabyte-source match.
```

Comment:
GH is horrible for PRs, but this content was just moved around, rather than net new addition. Note, there's a subsection below on `Replication lag`, which would use the `./determine_repl_latency.sh` script and examine the actual HT for an accurate and semi-automated view of the lag.